docs/source/daml-off-ledger.rst

```diff
@@ -0,0 +1,31 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Write Off-Ledger Automation Using Daml
+======================================
+
+The Daml smart contract language is mostly meant to provide a way to define on-ledger logic, i.e.
+code that defines how a transaction happens on ledger. Daml is not meant to be used as a general
+purpose language that can interact with your file system or network; instead, the templates and
+choices defined with Daml are available to be used by off-ledger logic that interacts with the
+ledger API. Usually this off-ledger logic is written in general-purpose language like Java or
+JavaScript and the codegen allows to interact with models defined in Daml without boilerplate.
```

Comment:
@carrielaben-da I rephrased "interact with Daml types". Does this make more sense?

Reply:
Yes, that makes sense.